examples/amp-story/analytics.html

```diff
@@ -104,15 +114,43 @@ <h1>Page Two</h1>
       <amp-story-page id="p3">
         <amp-story-grid-layer template="vertical">
           <h1>Page Three</h1>
+          <a href="google.com">click me</a>
```

Comment:
I'm not sure that I understand. But in your example, you are listening the `story-focus/story-click-through` event of all anchor tag. Do all anchor tag fire these two events, or it is only anchor tag created by embedded components and affiliate links? 

Would this anchor tag fire those two events? Thanks

Reply:
Only anchor tags created by embedded components and affiliate links will fire this event, since they are the only type of anchor tags that will show a tooltip in stories currently

Reply:
Yuxuan makes a good point here, what prevents this link within a grid-layer to trigger the regular `click` analytics event? Since it has custom events, it should not trigger this default "click". Wdyt?

Reply:
Sounds good, I added a `stopPropagation()` where appropriate so that these will only be handled by our custom events, and not by the regular `click` analytics event.